src/bundle/toolbar/types.ts

```diff
@@ -0,0 +1,23 @@
+import type {ActionStorage} from 'src/core';
+import type {ToolbarListButtonData} from 'src/toolbar';
```

Comment:
Plz, use relative path in imports
Because, builded types is generated in `build` directory, not in `src`